frappe/public/js/frappe/form/footer/timeline.js

```diff
@@ -109,6 +125,30 @@ frappe.ui.form.Timeline = class Timeline {
 			});
 	}
 
+	set_active_email_link() {
+		frappe.realtime.on('email_link', (message) => {
```

Comment:
Get the email via frappe.call, set it to `frappe.email.automatic_email_link`, show this value in timeline